### ISSUE TITLE
How to generate flems.html (static web app) from from a flems.io link?

### ISSUE THREAD

**Issue by starpause** [porsager/flems#64]
Is there a way to convert a flems.io link to a static (self-hosted) flems.html playground? For example, how would I convert 

https://flems.io/#0=N4IgzgpgNhDGAuEAmIBcIB0ALeBbKIANCAGYCWMYaA2qAHYCGuEamO+RIsA9nYn6wA8WAIwA+ADp0ABNIAqWMmGlLpAMRi4wUwQHpRkuoKRkAbiqQBeCaU3aQhvSdOOADodkaIWlcogMwCgBPaW8AI2QkZBUZJG5YAFdmPgZ4Ml5Q+FgMHJ1ddx0wWAAnMld4aTBi2GsQHHhXMFRdXToIAA94DBI7DHTdHu8wbDwoG2l4INcIWsRO3QArBlMAkrL4cdgsBmLIeFqE+BIAWgAOGzE9ItLyx2v1sWkpWVkAdzI6ONeMLy0ACkGWkI0mAzxe4PIlFQ0loYPB8OkjGY0IA5AxXK4MAswCjCHCES8eHwIHxUbgMMUSVFin8JFI6fA4olkl0wtwkEE8fSJHhaXURDZgTYABLQKDcaSvbjFKBIGwASjpdHlKPx4IAvgBdPEyAlQD4AayaMNBuoJLyREDJZHgWFKUFxaoJk2mqOxjrN5ukCRlqPqjWaugSdFcBoA5hgeLhdLgbXaKKrPRrNfj1YqjLp7rcQOrCPQmCx0FiqMQifx4Kwc3mQJbWJGwCWuLxy6xRMDXCCwbgdmGPsd4NxXNCAMwABlc7QA3GC5vBjgx9WG6NDYCTEMVp7qSM3jiQmMFoWAGHQwMdIKUSJv1fS6ABiQHKU2ybvFXvL6QAJnH7WkABZv5ushYBAZBhjg0L-gBYJsu0Z7bF80KjtISFfhO0ivmEDB-KOhA4ThGDDgArOm6o5pq6pAA

to 

```
<script src="https://flems.io/flems.html" type="text/javascript" charset="utf-8"></script>
<script>
Flems(document.body, {
    files: [{
        name: 'app.js',
        content: 'm.render(document.body, m("h1", "Hello world"))'
    }],
    links: [{
        name: 'mithril',
        type: 'script',
        url: 'https://unpkg.com/mithril'
    }]
})
</script>
```

So that I can embed it in my offline documentation =)

**Comment by tormodvm** [comment]
I mashed together something to convert a flems url into embed code.
Was it [something like this](https://flems.io/#0=N4Igxg9gdgzhA2BTEAucD4EMAONEBMQAaEGMAJw1QG0AGI2gXRIDMBLJGG0KTAW2RoAdAAsALn3jF0UMYlmoQAHhYRyfAATQYAVwBGfNmIC8AHRB9MAa0QAZNlCsAKAJQBucojE7yUDS0x4PHMAPlM-DSUHbB0xDWwsMEQRBHxEcjMQADEkPhgNH3hQ8I1IvVixaA0xAE9sREzdAyNiiNKAWWtEDUQ+PQINSDSSyIB6crFKqBDS8KVR1XUwqCU5AA8xTE9MEPn1ze2Q6TwkMDE2bUUAVhQAdhAAXyIefkEQIQArLhJIWXkxRQsHRQM4XPyWGz2RyuYAlX4wOKFDTGDT4CBgHQCWRCACOOnSNQAyohTpVyE4AOTRWIUlxCABugXxcO0cRYuRgAGEIGlkQVyPAhDAEkZKezejAhBdRgBiWjGWnUACMjA0AB81fyin54XEht0UQApQkAeQAckJsFs8E5bAAtQlicgOADmQjSkD42E8MBgWUofAAoiCeQQAKoAJQAktyvdB-k5xXluWkXC4WWkhOxOEJLNhExwDTNgBoRqVs4ghL85LI+RWq9Aa2IhJ4Epgkk5RkpTKZRmRndgxCFRmwXUQKd2e73+2xByFaSMHumIiU0Risc28QTiaS1JT9ltEJhaUJ9ty-rWUQADJQzwcaGDkMCZcRiXAoUYLDlSiBfiWiCQihAao6gacx9lGD5MEZO8xHMQYRGtLxMliFgAFoAA5QnmWDllvChZyHcIcglJw10xf4hD0HkaiIDQABJgGNc0hSdV02BYGonH1OioB0eB4DogAmFwlzmPsCLnK9wgeY4SUQUFLjQJUhJQJVHmeEBeAERQq19aRq3+RQD22DRYQiSxyBdBw0MqbAUA0ABmWhsDWNxSwiAB3Nh8DEEQHIANloABSNwRhERBR3EByhKuFy3JkuTSTBLhlLUgBOR5mBAeAHCsVLqBeHS0HgAAvNCEWdKAXWkQpFFfd9P2BbArDdT1RjKiq2OqgABJUhAAFkGjq2D0GAOvKyrXU+b4QFqepFFgrKSG0t4SLyLUNEAZAIZl6fp8EGUNHiAA) you had in mind? :)

**Comment by starpause** [comment]
That's awesome! Appreciate the quick solution.

**Comment by starpause** [comment]
Ran into a case that breaks your example, having `</script>` in a .html file of the flems. When I try and run the embed after copying it out of the embed-maker, the tags look like `<\/script>` in the .html preview. 

I can see that overloaded escape is happening intentionally with `file.content = file.content.replace(/<\/script>/ig,'<\\/script>')` but I'm not sure how to fix it. 

Without that line, `</script>` shows up in the `content` of the `.html` file, and when an unescaped / is hit there is an "unterminated string literal" javascript error. So I know why the double escaping is necessary, but not why it isn't double un-escaped on flems load or if there's another escaping technique which may work around this. 

TIA for any insight or help!

**Comment by tormodvm** [comment]
Yeah, I guess I jumped the gun there.
It's a bit tricky, since it is interpreted as the end of the current script tag.

Think I solved it with an ugly hack.  [Try this.](https://flems.io/#0=N4Igxg9gdgzhA2BTEAucD4EMAONEBMQAaEGMAJw1QG0AGI2gXRIDMBLJGG0KTAW2RoAdAAsALn3jF0UMYlmoQAHhYRyfAATQYAVwBGfNmIC8AHRB9MAa0QAZNlCsAKAJQBucojE7yUDS0x4PHMAPlM-DSUHbB0xDWwsMEQRBHxEcjMQADEkPhgNH3hQ8I1IvVixaA0xAE9sREzdAyNiiNKAWWtEDUQ+PQINSDSSyIB6crFKqBDS8KVR1XUwqCU5AA8xTE9MEPn1ze3l6TwkMDE2bUUAdhRaEABfIh5+QRAhACsuEkhZeTFFFg6KBnC5+Sw2eyOVzAEo-GBxQoaYwafAQMA6ASyIQARx06RqAGVEKdKuQnAByaKxckuIQAN0CeNh2jiLFyMAAwhA0kiCuR4EIYAkjBS2b0YEILqMAMS0Yw06gARkYGgAPqq+UU-Eg4kNusiAFIEgDyADkhNgtngnLYAFoEsTkBwAcyEaUgfGwnhgMCylD4AFFgdyCABVABKAEkuZ7oH8nGK8ly0i4XMyeYaTeb4U6oM62Cwak49UQNFAdPB4KWAExpvx63l6oSeBKYJJOUZKUymUZkJ3YMQhUZsZ1EcldkAAanM3d7FDYA5CNJGJVR6MxYhxePIhOJiDOagp+y2iEwNKE+y5v1kvIABko+wu4jByGBMuIxLgUKMFuzJRBf3FUQJCKEBqjqBpzH2UZ3kwBlHwHcxBhEK0vEyWIWAAWgADlCeYEMHOYCOWHJxScNcMT+IQ9G5GpSwAEmAPV7jrfD50XW9wnuY49xBS40EVRUUEVB4nhAXgBEUIQwB9aQfjkBQ0GPbYNBhCJLHIfMoEwypsBQDQAGZaGwNY3A0EYAHc2HwMQRH0gA2WgAFI3BGEREBHcR9OrABWYzTK4niSVBLgBOEgBOB5mBAeAHCsULqGeSS0HgAAvTCcxdaRCkUD8vx-IFsCsV0PVGNKMsdF0AAFFSEAAWeqyrYPQYDK9LMrzD4vhAWp6kUAiopICTXlIvJNQ0QBkAhmXp+nwQYQweIA)
